lib/oxigraph/src/store.rs

```diff
@@ -1710,6 +1712,15 @@ impl BulkLoader<'_> {
         }
     }
 
+    /// If `true` (the default), save to database only at the end instead of doing
+    /// it regularly while loading.
+    ///
+    /// The default value is `true`.
+    pub fn with_atomicity(mut self, atomicity: bool) -> Self {
```

Comment:
If the default is `true` why not making this method `without_atomity(mut self) -> self` to avoid the ambiguity on the default behavior?

Reply:
done